apps/backend/tests/test_services/test_fix_suggestion_engine.py

```
@@ -0,0 +1,376 @@
+                        transformation_type='trim_whitespace',
+                        parameters={'columns': ['name']},
+                        explanation='Trim whitespace',
+                        is_safe=True,
+                        confidence_score=0.95,
+                        estimated_rows_affected=3,
+                    )
+                ]
+            ),
+        ]
+
+        result_df, applied_fixes, errors = await engine.apply_batch_fixes(
```

Ruff (0.14.8)

Unpacked variable `errors` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)